staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5656,6 +5719,24 @@ type NodeDaemonEndpoints struct {
 	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty" protobuf:"bytes,1,opt,name=kubeletEndpoint"`
 }
 
+// RuntimeClassFeatures is a set of runtime features.
+type RuntimeClassFeatures struct {
```

Comment:
I get the need for this, but it almost feels like a different feature - do we need it for RRO?

Reply:
The E2E test suite depends on this to detect whether the node supports RRO

Reply:
I'm personally not a huge fan of adding it just for the tests. Can we detect whether the node supports by parsing the response to a create or something like that? Or wait until CRIs support the field and then add e2e tests?

Reply:
Not just for tests. Useful for human users too. This was already discussed in the original KEP, but hard to find the link to the thread.